doc/release-notes.md

```diff
@@ -19,24 +19,16 @@ To receive security and update notifications, please subscribe to:
 Compatibility
 ==============
 
+Bitcoin Core is extensively tested on multiple operating systems using
+the Linux kernel, MacOS x.y+, and Windows later than XP.
+
 Microsoft ended support for Windows XP on [April 8th, 2014](https://www.microsoft.com/en-us/WindowsForBusiness/end-of-xp-support),
-an OS initially released in 2001. This means that not even critical security
-updates will be released anymore. Without security updates, using a bitcoin
-wallet on a XP machine is irresponsible at least.
-
-In addition to that, with 0.12.x there have been varied reports of Bitcoin Core
-randomly crashing on Windows XP. It is [not clear](https://github.com/bitcoin/bitcoin/issues/7681#issuecomment-217439891)
-what the source of these crashes is, but it is likely that upstream
-libraries such as Qt are no longer being tested on XP.
-
-We do not have time nor resources to provide support for an OS that is
-end-of-life. From 0.13.0 on, Windows XP is no longer supported. Users are
-suggested to upgrade to a newer version of Windows, or install an alternative OS
-that is supported.
-
-No attempt is made to prevent installing or running the software on Windows XP,
-you can still do so at your own risk, but do not expect it to work: do not
-report issues about Windows XP to the issue tracker.
+an OS initially released in 2001. No attempt is made to prevent installing or running the software on Windows XP,
+you can still do so at your own risk. Please do not report issues about Windows XP
```

Comment:
Maybe mention that there are known instabilities/bugs on XP, which are not going to be fixed.